architecture/2-UserStories/2-ViewBeaconData.yaml

```diff
@@ -0,0 +1,15 @@
+id: 2
+name: View Satellite Beacon Data
+actor: Ground Station Operator
+behavior: view satellite beacon data (alternating between health & mission) data, received via UHF
```

Comment:
Reads "data (alternating between health & mission) data" here

Reply:
Yeah, the idea is that the beacon data will alternate between transmitting both health data and mission data, as to offload having to request mission data manually. Though, the verbiage could use some refining

Reply:
Or were you suggesting the 2nd "data" is a bit redundant?

Reply:
I read it as "satellite beacon data data" if you removed the part in parenthesis with the part in parenthesis describing "satellite beacon data".  It looks like you just repeated "data". Place the second "data" within the parenthesis